python/triton/language/extra/libdevice.py

```diff
@@ -0,0 +1,1213 @@
+from .cuda import libdevice as cuda_libdevice
+from .hip import libdevice as hip_libdevice
+from triton.language import core
+from functools import wraps
+from typing import TypeVar
+
+T = TypeVar('T')
+
+
+def dispatch(fn: T) -> T:
```

Comment:
are those files auto-generated? If so could you indicate how to re-generate them?

Reply:
> are those files auto-generated? 

Hi @ThomasRaoux,
No, it is not auto-generated